app/models/progress_message.rb

```diff
@@ -5,6 +5,12 @@ class ProgressMessage < ApplicationRecord
 
   AS_JSON_ATTRIBUTES = %w(id level message received_at).freeze
 
+  scope :by_owner, lambda {
```

Comment:
Small nit. Let's change this to the newer style `->` vs. `lambda`.

Reply:
rubocop complains about it though :(